index.html

```diff
@@ -21,6 +24,12 @@ <h1>Lucas Caldas da Costa | WDD130</h1>
             Biomedical Sciences Student; TRPG Enthusiast and Amateur Tabletop Game Designer; Rio de Janeiro, Brazil; Enjoys eating anything tasty that doedn't include coffee.
             </p>
         </main>
+        <aside>
+            <h2>Rio de Janeiro, Brazil</h2>
+            <img src="images/Bandeira_do_estado_do_Rio_de_Janeiro.svg.png" alt="Rio de Janeiro State Flag">
+            <img src="images/bandeira_do_brasil.jpeg" alt="Brazil Flag">
```

Comment:
This image was not found. Please, remove this line from your index.html.

Remove line 30.